doc/install-on-docker.md

```diff
@@ -0,0 +1,67 @@
+# Installation on Docker
+
+You can install Ray on any platform that runs Docker. We do not presently publish Docker images for Ray, but you can build them yourself using the Ray distribution. Using Docker can provide a reliable way to get up and running quickly.
+
+## Install Docker
+
+The Docker Platform release is available for Mac, Windows, and Linux platforms. Please download the appropriate version from the [Docker website](https://www.docker.com/products/overview#/install_the_platform).
+
+## Clone the Ray repository
+
+```
+git clone https://github.com/amplab/ray.git
+```
+
+## Build Docker images
+
+Run the script to create Docker images.
+
+```
+cd ray
+./build-docker.sh
+```
+
+This script creates several Docker images:
+
+ * The `amplab/ray:deploy` image is a self-contained copy of code and binaries suitable for end users.
+ * The `amplab/ray:examples` adds additional libraries for running examples.
```

Comment:
It's probably worth it to (eventually) push these images to amplab's Dockerhub so the user doesn't have to build them.